### PR TITLE
BFD-591: Update NDC Code and Display for Tests

### PR DESCRIPTION
**Pull request by jzulim**
### Change Details

- This PR proposes a change to our drug code tests so that they pass again. The previous drug and code is no longer within the NDC product list. I've selected a new drug (Tylenol) to use for all NDC related tests. 

### Acceptance Validation
- IT's begin to pass again. 

### Feedback Requested
- The drug selected for tests is a good one? 

### External References

- [BLUEBUTTON-591](https://jira.cms.gov/browse/BLUEBUTTON-591)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

File: apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-static-samples/sample-a-dme.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
-INSERT|567834|2188888888|900|F|M|82|03-FEB-2014|03-FEB-2014|14-FEB-2014|1|01|99999|1|777.75|0|A|666.75|666.66|1752.75|754.79|777.00|3|R5555|0|R5555|0|R6666|0|||||||||||||||||||||I99999|1306849450|0|74655592568216|1|9994931888|A5|1|60|P|12|03-FEB-2014|03-FEB-2014|345|YY||D9Z|123.45|11.00|120.00|18.00|E|11.00|20.20|20.29|130.45|129.45|A|0|0|G6666|0|82.29|1219966666|1244444444|AL|MO|3|||0.00|60|3|44.4|R2|665561945|
+INSERT|567834|2188888888|900|F|M|82|03-FEB-2014|03-FEB-2014|14-FEB-2014|1|01|99999|1|777.75|0|A|666.75|666.66|1752.75|754.79|777.00|3|R5555|0|R5555|0|R6666|0|||||||||||||||||||||I99999|1306849450|0|74655592568216|1|9994931888|A5|1|60|P|12|03-FEB-2014|03-FEB-2014|345|YY||D9Z|123.45|11.00|120.00|18.00|E|11.00|20.20|20.29|130.45|129.45|A|0|0|G6666|0|82.29|1219966666|1244444444|AL|MO|3|||0.00|60|3|44.4|R2|667159747|
```
Comment (cbrunefearless) [inline apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-static-samples/sample-a-dme.txt:2]: Good stuff!!

File: apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/PartDEventTransformerTest.java [hunk 2]
```diff
@@ -283,7 +283,7 @@ static void assertMatches(PartDEvent claim, ExplanationOfBenefit eob) throws FHI
     TransformerTestUtils.assertLastUpdatedEquals(claim.getLastUpdated(), eob);
     try {
       TransformerTestUtils.assertFDADrugCodeDisplayEquals(
-          claim.getNationalDrugCode(), "Hand E San - ALCOHOL");
+          claim.getNationalDrugCode(), "TYLENOL EXTRA STRENGTH - ACETAMINOPHEN");
```
Comment (cbrunefearless) [inline apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/PartDEventTransformerTest.java:286]: Better pick than mine

**Review by cbrunefearless — APPROVED** [review]
LGTM

**Review by inChargeOfIT — APPROVED** [review]
LGTM

**Review by keithdadkins — APPROVED** [review]
LGTM

**Pull request merged**